symmer/projection/contextual_subspace.py

```diff
@@ -136,6 +185,31 @@ def _random_stabilizers(self,
         
         return S
 
+    def _get_clique_representatives(self, symmetry_terms=None, n_cliques=2, clique_reps = []):
```

Comment:
This method identifies a list of anticommuting operators that commute with the selected stabilizers (the required noncontextual structure). It is also possible to force stabilizers within a clique by starting recursion with clique_reps non-empty.